Origin: upstream, https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/?id=dddf1b44126eaa25c6698be618fb452fc7b9bed3
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1800877
From: Stephen Hemminger <shemming@brocade.com>
Date: Fri, 23 Oct 2015 15:47:07 -0700
Subject: [PATCH] add new IFLA_VF_TRUST netlink attribute

---
 include/linux/if_link.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/linux/if_link.h b/include/linux/if_link.h
index 288d3cd61679..31f584cb2576 100644
--- a/include/linux/if_link.h
+++ b/include/linux/if_link.h
@@ -500,6 +500,7 @@ enum {
 				 * on/off switch
 				 */
 	IFLA_VF_STATS,		/* network device statistics */
+	IFLA_VF_TRUST,		/* Trust VF */
 	__IFLA_VF_MAX,
 };
 
@@ -561,6 +562,11 @@ enum {
 
 #define IFLA_VF_STATS_MAX (__IFLA_VF_STATS_MAX - 1)
 
+struct ifla_vf_trust {
+	__u32 vf;
+	__u32 setting;
+};
+
 /* VF ports management section
  *
  *	Nested layout of set/get msg is:
-- 
2.17.1

